frontend/src/app/modules/players/players-selectors.js

```diff
@@ -62,3 +62,18 @@ const getPlayerCriteriaPoints = (player, matchesLast, criteria) => {
       return player.rating
   }
 }
+
+export const getRankingsForPlayer = createSelector(
```

Comment:
The same as in the message bellow...

Reply:
Answered below if it was about undefined transform function in selector problem

Reply:
Sorry, I meant conversation about [this](https://github.com/salsita/foosball-rating/pull/114/files/bf6e918f3cf0c320e1efcf81fab8d2d8d76e9025#diff-cd4220e6543e26c82876c33466c868e0) file but nevertheless, I read something about the problem and as I understand it there will be no such issue. There is only one instance of player statistics at the time and situation where multiple such would exist seems pretty exotic to me. Please correct me if I am wrong.

Reply:
agree it's exotic on this page, but if ProfileStatistics would be connected standalone component and could be used twice somewhere else then it would be the problem. For now it's ok then